src/server/terrain/simple_structures/SBBGen.zig

```diff
@@ -0,0 +1,122 @@
+const std = @import("std");
```

Comment:
file name should be sbb_gen according to the name guidelines

Reply:
Well, either sbb_gen or SbbGen, can't be both, I assume latter since this is a file struct.

Reply:
Yeah, if it's a struct then SbbGen